contracts/RenPool.sol

```diff
@@ -1,10 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.7;
 
+import "hardhat/console.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "../interfaces/IDarknodeRegistry.sol";
 import "../interfaces/IDarknodePayment.sol";
-import "../interfaces/IClaimRewards.sol";
+import "../interfaces/IClaimRewardsV1.sol";
```

Comment:
@acuarica not sure if we want to call this `ClaimRewardsV1` or simple `ClaimRewards`

Reply:
We can use `IClaimRewardV1` to make explicit that we are using that version. Also once this API is stabilized we might want to add its respective check to `test/ren`.

Reply:
Ok, I'll make that change :+1: